README.md

```diff
@@ -22,8 +22,10 @@ are contained in the 'res' directory.
 The following programs and/or libaries need to be installed before compiling
 and installing the RMA:
 
-Asterisk (>= version 13.0.0)
-libcurl
+- Asterisk (>= version 13.0.0)
+- libcurl
+- pjproject
+- res_rtp_asterisk
```

Comment:
Yes, you can build res_rtp_asterisk without pjproject. chan_sip uses res_rtp_asterisk, as do (most) other RTP capable channel drivers.